src/hotspot/share/code/aotCodeCache.hpp

```diff
@@ -300,9 +400,12 @@ class AOTCodeCache : public CHeapObj<mtCode> {
   void load_strings();
   int store_strings();
 
-  static void init_early_stubs_table() NOT_CDS_RETURN;
-  static void init_shared_blobs_table() NOT_CDS_RETURN;
-  static void init_early_c1_table() NOT_CDS_RETURN;
+  static void set_shared_stubs_complete() NOT_CDS_RETURN;
```

Comment:
I don't see `set_shared_stubs_complete`, `set_c1_stubs_complete` and `set_c2_stubs_complete` called from anywhere.